src/EnergyPlus/api/datatransfer.cc

```diff
@@ -239,6 +240,22 @@ void resetErrorFlag(EnergyPlusState state)
     thisState->dataPluginManager->apiErrorFlag = false;
 }
 
+char *inputFilePath(EnergyPlusState state)
+{
+    const auto *thisState = static_cast<EnergyPlus::EnergyPlusData *>(state);
+    char *p = new char[std::strlen(thisState->dataStrGlobals->inputFilePath.string().c_str()) + 1];
+    std::strcpy(p, thisState->dataStrGlobals->inputFilePath.string().c_str());
+    return p;
+}
+
+char *epwFilePath(EnergyPlusState state)
+{
+    const auto *thisState = static_cast<EnergyPlus::EnergyPlusData *>(state);
+    char *p = new char[std::strlen(thisState->files.inputWeatherFilePath.filePath.c_str()) + 1];
+    std::strcpy(p, thisState->files.inputWeatherFilePath.filePath.c_str());
+    return p;
+}
```

Comment:
Added a similar method for getting the epw.

Reply:
One is doing `fs::path.string().c_str()` and the other is doing `fs::path.c_str()`